src/MessageContainer.js

```diff
@@ -109,13 +117,15 @@ export default class MessageContainer extends React.Component {
       message.user = {};
```

Comment:
I will check all changes over the weekend and provide some screens (and improve style of system message).